README.md

```diff
@@ -125,6 +125,15 @@ def hook(state):
 m.run()
 ```
 
+### Ethereum
```

Comment:
i feel like the readme is getting kinda bloated and shouldn't be used for manticore's documentation. I think we should create a docs/ page for the cli and move this there. on the readme we can have a little teaser above alongside the `/path/to/binary` lines. let's also edit the below "API reference" line; that site is no longer just the API reference, it's manticore's documentation.